tests/travis/_config.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
Check https://github.com/silverstripe/silverstripe-blog/blob/master/.travis.yml for an example on how we normally do this.